PartsApp/Form1.cs

```diff
@@ -1168,7 +1166,7 @@ private void Form1_FormClosed(object sender, FormClosedEventArgs e)
         {
             //Если Титор, тогда делаем бэкап (Для проверки на Lock conflict)
             if (CurEmployee?.EmployeeId == 1)
```

Comment:
This is a bad approach. Create a US somewhere for refactoring to change this and to come up with another backup mechanism.

Reply:
done
https://dev.azure.com/razor23donetsk0618/TruckTir/_workitems/edit/41